### DISCUSSION TITLE
Write the names of the translator contributors for each of their local languages

### DISCUSSION THREAD

**Discussion by avirahmandev** [godotengine/godot-proposals#9820] [General Discussions]
Write the name of the translator for each of their local languages ​​inside godot about page.

![godot-authors](https://github.com/godotengine/godot-proposals/assets/50703840/e58e7e15-bd9b-4fd7-83ae-26a6faf75102)

**Comment by AThousandShips** [comment]
How would contributions be counted and what would be the threshold for inclusion? Other contributors are included only with over 10 contributions

**Reply by avirahmandev** [reply]
Can't you just use existing data from weblate?

For the inclusion threshold, the top order 20% of contributions from all contributors, maybe?

**Reply by AThousandShips** [reply]
> Can't you just use existing data from weblate?

That's what I asked, that would make sure, but it should be a specific number of contributions to be fair

**Reply by avirahmandev** [reply]
what is the specific number contributions? 

why don't just write down all the names, I don't know what meaning is to be fair here

**Reply by AThousandShips** [reply]
> what is the specific number contributions?

For code contributions it is 10, but I asked how it would be computed here because I'm not sure

But I don't think just a single contribution should get you named, it should have a minimum requirement to be fair with everything else

Why should someone translating a single sentence be included but not someone making 9 code contributions?

**Reply by avirahmandev** [reply]
Understood, I think it's the internal team job to make that decision, I'm just submitting a proposal regarding that, and if I mention specific numbers contributions it's still a consideration for the internal team, right? so for decision making I leave it to the godot team, like 10 code contributions. It doesn't seem fair if follow my subjective opinion

**Reply by timothyqiu** [reply]
For po files itself, there's no way to count a translator's contribution. It only has a list of users that ever touched the translation.

Weblate does not provide this kind of statistics either I believe.

**Reply by AThousandShips** [reply]
Also, to be clear, I think it's important to acknowledge all contributions, including translation, and the reason I'm asking these questions is to try to make it a bit clearer how this might be hard (and even why it hasn't already been done)

I would suggest, to make it less cluttered in the "About" section, to come up with a way to count contributions for translations (in all languages, if people work with multiple ones) and then add translators of the *engine* translations (i.e. not documentation etc. but just editor interface etc.) to the normal contributors list, that way we don't have to work with more complicated UI, and we don't have to figure out how to handle people being added twice.

**Reply by avirahmandev** [reply]
> I would suggest, to make it less cluttered in the "About" section, to come up with a way to count contributions for translations (in all languages, if people work with multiple ones) and then add translators of the _engine_ translations (i.e. not documentation etc. but just editor interface etc.) to the normal contributors list, that way we don't have to work with more complicated UI, and we don't have to figure out how to handle people being added twice.

Agree,

Responding to @timothyqiu's comment, what are the next steps to possibly get this data?
